packages/tasks/src/model-libraries-snippets.ts

```diff
@@ -170,6 +170,41 @@ export const diffusers = (model: ModelData): string[] => {
 	}
 };
 
+export const diffusionkit = (): string[] => {
+	const sd3Snippet = `from diffusionkit.mlx import DiffusionPipeline
+
+pipeline = DiffusionPipeline(
+	shift=3.0,
+	use_t5=False,
+	model_version="argmaxinc/mlx-stable-diffusion-3-medium",
+	low_memory_mode=True,
+	a16=True,
+	w16=True,
+)`;
+	const fluxSnippet = `from diffusionkit.mlx import FluxPipeline
+
+pipeline = FluxPipeline(
+  shift=1.0,
+  model_version="argmaxinc/mlx-FLUX.1-schnell",
+  low_memory_mode=True,
+  a16=True,
+  w16=True,
+)`;
+	const generateSnippet = `HEIGHT = 512
+WIDTH = 512
+NUM_STEPS = 4  #  4 for FLUX.1-schnell, 50 for SD3
+CFG_WEIGHT = 0. # for FLUX.1-schnell, 5. for SD3
+
+image, _ = pipeline.generate_image(
+  "a photo of a cat",
+  cfg_weight=CFG_WEIGHT,
+  num_steps=NUM_STEPS,
+  latent_size=(HEIGHT // 8, WIDTH // 8),
+)`;
+
+	return [sd3Snippet, fluxSnippet, generateSnippet];
```

Comment:
Since we have multiple snippets, it may be useful to include a comment line at the beginning of each one to explain what they are about.

Reply:
Agreed and added, thanks!